Sources/Crypto/AEADs/AES/GCM/AES-GCM.swift

```diff
@@ -11,10 +11,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 //===----------------------------------------------------------------------===//
-#if (os(macOS) || os(iOS) || os(watchOS) || os(tvOS)) && CRYPTO_IN_SWIFTPM && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
+#if canImport(CryptoKit) && CRYPTO_IN_SWIFTPM && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
 @_exported import CryptoKit
 #else
-#if (os(macOS) || os(iOS) || os(watchOS) || os(tvOS)) && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
+#if canImport(CryptoKit) && !CRYPTO_IN_SWIFTPM_FORCE_BUILD_API
```

Comment:
This is wrong: these files are used to _build_ `CryptoKit`, so this branch can't safely rely on the `canImport` here. Prefer `canImport(Darwin)`.